binsync/common/ui/force_push/panels/functions_table.py

```diff
@@ -0,0 +1,483 @@
+import logging
+from datetime import datetime
+from typing import Dict
+
+from binsync.common.controller import BinSyncController
+from binsync.common.ui.qt_objects import (
+    QAbstractItemView,
+    QAbstractTableModel,
+    QHeaderView,
+    QMenu,
+    Qt,
+    QModelIndex,
+    QSortFilterProxyModel,
+    QColor,
+    QFocusEvent,
+    QKeyEvent,
+    QLineEdit,
+    QTableView,
+    QFontDatabase,
+    QAction,
+    QWidget,
+    QVBoxLayout,
+    QPushButton,
+    QPersistentModelIndex,
+    QCheckBox
+)
+from binsync.common.ui.utils import friendly_datetime
+from binsync.data import Function
+from binsync.data.state import State
+from binsync.core.scheduler import SchedSpeed
+
+l = logging.getLogger(__name__)
+
+
+class FunctionTableModel(QAbstractTableModel):
+    """Table model that controls backend behavior of the function table"""
+    HEADER = [
+        'Addr',
+        'Remote Name',
+        'User',
```

Comment:
Recall again most of this other data for a user that is pushing is useless. Why would I need to know how many times another user changes this function when I am force pushing.